extensions/jackson/deployment/src/test/java/io/quarkus/jackson/deployment/JacksonTimeZonePropertiesTest.java

```diff
@@ -25,7 +28,8 @@ public class JacksonTimeZonePropertiesTest {
 
     @Test
     public void testTimezone() throws JsonProcessingException {
-        Assertions.assertThat(objectMapper.writeValueAsString(new Pojo(new Date(2021, Calendar.MARCH, 3, 11, 5))))
+        Assertions.assertThat(objectMapper.writeValueAsString(new Pojo(Date.from(
+                ZonedDateTime.of(LocalDateTime.of(2021, Month.MARCH, 3, 11, 5), ZoneId.of("GMT")).toInstant()))))
```

Comment:
The zone here does not matter because the point of the test is to show that Jackson overrides the timezone with the one configured in `application.properties`